test.js

```diff
@@ -162,8 +162,42 @@ describe( "collect", function () {
     })
 })
 
+describe( "Fetch BOT", function () {
+    var result = {};
+    var source = {
+        apps: [ 'myApp' ],
+        ignoreMissing: true
+    }
+
+    var response = { "myApp": { error: {}, name: "myApp", data: dataGenerator( 1, null ) } }
+
+    before( initialize( result, response, source, true ) )
+    after( reset )
+
+    it( 'Skip missing data', function () {
```

Comment:
I am not sure what are you testing here and what it has to do with missing data?